soh/src/overlays/misc/ovl_kaleido_scope/z_kaleido_scope_PAL.c

```diff
@@ -3159,11 +3159,11 @@ void KaleidoScope_Update(GlobalContext* globalCtx)
     GameOverContext* gameOverCtx = &globalCtx->gameOverCtx;
     Player* player = GET_PLAYER(globalCtx);
     Input* input = &globalCtx->state.input[0];
-    size_t size;
-    size_t size0;
-    size_t size1;
-    size_t size2;
-    u16 i;
+    size_t size = 0;
+    size_t size0 = 0;
+    size_t size1 = 0;
+    size_t size2 = 0;
+    u16 i = 0;
```

Comment:
Is this needed?

Reply:
I'm guessing MSVC was complaining about uninitialized stack variables.